filecoin-proofs/src/bin/paramfetch.rs

```diff
@@ -117,14 +119,12 @@ fn fetch(matches: &ArgMatches) -> Result<()> {
     let retry = matches.is_present("retry");
     let gateway = matches.value_of("gateway").unwrap_or("https://ipfs.io");
 
-    if !json_path.exists() {
-        return Err(err_msg(format!(
-            "json file '{}' does not exist",
-            &json_path.to_str().unwrap_or("")
-        )));
-    }
+    let manifest = if json_path.exists() {
```

Comment:
Please do the following:

- If the user provides a value for `json` and it is not a path for a file that exists on the disk, produce an error
- If the user does not provide a value for `json`, use the default-JSON baked into the binary
- After this change merges, modify the `paramfetch` section [of the go-filecoin wiki](https://github.com/filecoin-project/go-filecoin/wiki/Getting-Started#installing-from-binary) to demonstrate pointing `paramfetch` at the JSON manifest using the `--json` arg

I've [already written some tests for `paramfetch`](https://github.com/filecoin-project/rust-fil-proofs/blob/79b92f77b96209e387630d77bfa124b71c8610aa/filecoin-proofs/tests/paramfetch/prompts_to_fetch.rs#L30). I'd really like it if you'd augment the exist test suite with:

- a test which verifies that `paramfetch` fails with a non-zero exit code if given a `json` value which is not a path to a file on disk
- a test which verifies that `paramfetch` fails if given a `json` value which does not contain JSON which it can deserialize to a `ParameterMap`
-  a test which verifies that `paramfetch`, when not provided a `json` argument, [checks](https://github.com/filecoin-project/rust-fil-proofs/blob/79b92f77b96209e387630d77bfa124b71c8610aa/filecoin-proofs/tests/paramfetch/prompts_to_fetch.rs#L75) for the existence of the files in the manifest